documentation/developer_guide/testing.rst

```diff
@@ -254,20 +254,20 @@ Add a new device
 
     mf2c-curl-post https://localhost/api/device -d '''
     {
-    'deviceID': 'fd97ac4cf865e108c143c57428f742022f38653f1f4c4166938a3154d7b5818967fd27dae6422a2b1da1ceb8dc9d25f3585ab7b4039c96b5d9ad43acb7dce0ff',
-    'isLeader': False,
-    'os': 'Linux-4.15.0-45-generic-x86_64-with-debian-9.7',
-    'arch': 'x86_64',
-    'cpuManufacturer': 'Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz',
-    'physicalCores': 4,
-    'logicalCores': 8,
-    'cpuClockSpeed': '1.8000 GHz',
-    'memory': 7873.7734375,
-    'storage': 195865.0234375,
-    'agentType': 'Fog agent',
-    'networkingStandards': "['eth0', 'lo']",
-    'hwloc': '/bin/sh: 1: hwloc-ls: not found\n',
-    'cpuinfo': 'xml info CPU'
+    "deviceID": "fd97ac4cf865e108c143c57428f742022f38653f1f4c4166938a3154d7b5818967fd27dae6422a2b1da1ceb8dc9d25f3585ab7b4039c96b5d9ad43acb7dce0ff",
+    "isLeader": False,
+    "os": "Linux-4.15.0-45-generic-x86_64-with-debian-9.7",
+    "arch": "x86_64",
+    "cpuManufacturer": "Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz",
+    "physicalCores": 4,
+    "logicalCores": 8,
+    "cpuClockSpeed": '1.8000 GHz',
+    "memory": 7873.7734375,
+    "storage": 195865.0234375,
+    "agentType": "Fog agent",
+    "networkingStandards": "["eth0", "lo"]",
```

Comment:
wrong quotes

Reply:
Can you please suggest me where is the wrong one.. is it in the `networkingStandards`

Reply:
`"networkingStandards": "["eth0", "lo"]",` this is segmented

Reply:
so it should be like - `"networkingStandards": "['eth0', 'lo']"`, right?

Reply:
> `"networkingStandards": "["eth0", "lo"]",` this is segmented

Oh! Yes.....I got it.... :(